bi/pkg/cluster/kind/net_test.go

```diff
@@ -0,0 +1,43 @@
+package kind
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_split(t *testing.T) {
+	testCases := []struct {
+		sub, expected string
+		existing      int
+		errExpected   bool
+	}{
+		{sub: "172.0.0.0/8", expected: "172.0.1.0/24", existing: 0},
```

Comment:
These tests were added with the existing functionality and not changed (except for adding the offset) so we're getting the same results with `cidr` as with `ipnets`.